coding-concepts.js

```diff
@@ -69,5 +70,5 @@ class Learn {
 var learnStudent = new Learn("George")
 // console.log(learnStudent)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: [student: George, cohort: Bravo, year: 2022]
+// b) Verify and explain: Learn { student: 'George', cohort: 'Bravo', year: 2022 }, this was the output because the new instance made the name parameter in the constructor method was now assigned to George I made the mistake of putting the class in [] instead of {}. 
```

Comment:
Keeping the awesomeness going as we move down the page!  You are showing high levels of concept understanding.